ui/app/sample-data/dashboard.ts

```diff
@@ -0,0 +1,273 @@
+import { DashboardResource, AnyVariableDefinition } from '@perses-ui/core';
+
+const cadvisor: DashboardResource = {
+  kind: 'Dashboard',
+  metadata: {
+    name: 'Kubelet - cAdvisor',
```

Comment:
The public demo server doesn't have Kubernetes data (nor does my PromLabs one), so it would be better to check in a Node Exporter dashboard or something that works against the data in that server.

Reply:
well the goal is more to use the json you have in the directory `./dev`. These data will just stay until the frontend is connected to the backend

Reply:
here it's just adding data to be able to build and start the app